mne/preprocessing/tests/test_ica.py

```diff
@@ -268,7 +274,16 @@ def test_ica_additional():
               n_pca_components=4)
     assert_raises(RuntimeError, ica.save, '')
     with warnings.catch_warnings(record=True):
-        ica.fit(raw, picks=None, start=start, stop=stop2)
+        ica.fit(raw, picks=[1, 2, 3, 4, 5], start=start, stop=stop2)
+
+    # test corrmap
+    ica2 = deepcopy(ica)
```

Comment:
do we have `ica.copy()`? If not, we probably should. If so, it should be used here instead.